enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -56,23 +49,10 @@ export const _InteractiveQuestion = ({
           withTitle={withTitle}
         />
       )}
-    </InteractiveQuestionProviderWithLocation>
+    </InteractiveQuestionProvider>
   );
 };
 
-export const getQuestionParameters = (questionId: CardId) => {
```

Comment:
We don't need this anymore - since users are already passing a `questionId`, we can just pass that into the location-agnostic provider and call it a day. When I was making this change, I also then realized that we can just use the `InteractiveQuestion` components in our `QuestionEditor`